### PR TITLE
Commenting on Journal To Do List widget

### LINKED ISSUE #225: Commenting on the Journal/Dashboard Todo widgets for code readability
**Is your feature request related to a problem? Please describe.**
As a code contributor, I would like the code to be well commented on so that it is easy to figure out how the code works for future contributors to be able to work on it. 

**Describe the solution you'd like**
I would like comments on both the Journal and Dashboard Todolist where the code is harder to understand

**Describe alternatives you've considered**
No comments, which would make it harder for future contributors to understand the code

### PR DESCRIPTION
**Pull request by cng822**
Closes #225 

## Proposed Changes

- Added detailed comments to help the understanding of the code to the JournalToDo.jsx
- Removed commented code that was still remaining from the previous pull requests and is no longer needed

### CODE REVIEW

**Review by AngelaLH — APPROVED** [review]
LGTM, the comments make the code much easier to understand :)

**Comment by yzho976** [comment]
This looks good, comments are really detailed enough. Just solve the eslint problem and it can be merged.

File: telos-frontend/src/components/todo/JournalTodo.jsx [hunk 1]
```diff
@@ -261,6 +249,8 @@ const JournalTodo = ({ data, date }) => {
         })}
       </List>
       <div>
+       { /* A textbox to indicate to the user that they can enter new to do's.
```
Comment (yzho976) [inline telos-frontend/src/components/todo/JournalTodo.jsx]: Here, a little bit change is needed.

File: telos-frontend/src/components/todo/JournalTodo.jsx [hunk 2]
```diff
@@ -269,7 +259,8 @@ const JournalTodo = ({ data, date }) => {
             label="Disabled"
             endAdornment={
               <InputAdornment position="end">
-                <IconButton
+              {/* When the user clicks on the button a modal will pop up allowing the user to set their requirements */}
```
Comment (yzho976) [inline telos-frontend/src/components/todo/JournalTodo.jsx]: Same here

**Review by yzho976 — APPROVED** [review]
Overall I think it can be merged once eslint problems are fixed.

File: telos-frontend/src/components/todo/JournalTodo.jsx (hunk 1 above)
Comment (cng822) [reply telos-frontend/src/components/todo/JournalTodo.jsx]: Hey Evan, thank you for the suggestion! I have removed the space inside the comment '  { /*  ' as the following change after your feedback :)

File: telos-frontend/src/components/todo/JournalTodo.jsx (hunk 2 above)
Comment (cng822) [reply telos-frontend/src/components/todo/JournalTodo.jsx]: I have aligned the code with the comment required

**Pull request merged**